apps/api/src/beneficiary/beneficiary.controller.ts

```diff
@@ -1,26 +1,58 @@
-import { Body, Controller, Get, Post } from '@nestjs/common'
+import { Body, Controller, Delete, Get, Param, Post, Put } from '@nestjs/common'
 import { Public, RoleMatchingMode, Roles } from 'nest-keycloak-connect'
 import { RealmViewSupporters, ViewSupporters } from '@podkrepi-bg/podkrepi-types'
 
 import { BeneficiaryService } from './beneficiary.service'
 import { CreateBeneficiaryDto } from './dto/create-beneficiary.dto'
+import { UpdateBeneficiaryDto } from './dto/update-beneficiary.dto'
 
 @Controller('beneficiary')
 export class BeneficiaryController {
   constructor(private readonly beneficiaryService: BeneficiaryService) {}
 
   @Post('create-beneficiary')
-  @Public()
+  @Roles({
+    roles: [RealmViewSupporters.role, ViewSupporters.role],
+    mode: RoleMatchingMode.ANY,
+  })
   async create(@Body() createDto: CreateBeneficiaryDto) {
     return await this.beneficiaryService.createBeneficiary(createDto)
   }
 
   @Get('list')
+  // @Roles({
+  //   roles: [RealmViewSupporters.role, ViewSupporters.role],
+  //   mode: RoleMatchingMode.ANY,
+  // })
```

Comment:
If we need this code to effective please remove the comment, if we don;t need this code please remove the code.

Not sure what should be the exact action here, maybe de-comment

Reply:
This is for authentication.

Reply:
Fixed.